spec/01-lexical-syntax.md

```diff
@@ -553,6 +553,22 @@ but are required to be properly nested.  Therefore, a comment like
 `/* /* */` will be rejected as having an unterminated
 comment.
 
+## Multi-line and trailing commas
+
+Trailing commas are allowed when followed by a newline and a closing
```

Comment:
I suggest bringing this closer to the implementation, which ignores a (single) comma if the following token is a right-parenthesish that itself comes after a line end.